docs/site/tutorials/protocol_oriented_generics.ipynb

```diff
@@ -0,0 +1,1141 @@
+{
```

Comment:
<p>This section is set up as a negative. (Basically, we are telling the reader that what they know and love is wrong.) Is there a way we can take the ideas you have in this section, and make them positive? Maybe a narrative arc along the lines of:</p><ul><li>Swift has value types.</li><li>What are value types?</li><li>Why are value types awesome?</li></ul><p>And then include these advantages in that section.</p><p><br></p><p>Does that make sense?</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/swift/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>